mta/conf/env.yaml

```diff
@@ -1,6 +1,9 @@
 application:
   hostname: http://localhost:8080
+  ocphostname: http://mta.apps.sshveta.f2aa.sandbox1745.opentlc.com
+  ocpsecurehostname: https://secure-mta.apps.sshveta.f2aa.sandbox1745.opentlc.com/
```

Comment:
For now, it's ok.
In the future, we can handle this in application class only by taking. I would take the hostname filed only

```
hostname: foo
scheme: http/https
secure: flase/true
password:
```
and handle URL in class only.